Templates/AzureDevOpsPipeline/templates/README.md

```diff
@@ -0,0 +1,13 @@
+# README
+
+This folder contains [Azure DevOps Templates](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops&pivots=templates-includes) yaml configurations, that are referenced by the core Azure pipeline. 
+
+Please work with your Azure DevOps administrators where to maintain these scripts. 
```

Comment:
'Please work with your Azure DevOps administrators where to maintain these scripts.'
'Please work with your Azure DevOps administrators to agree on how and where to maintain these scripts.'